bin/propolis-server/src/lib/vm/mod.rs

```diff
@@ -347,50 +334,6 @@ impl Vm {
         .ok()
     }
 
-    /// Returns the state, properties, and instance spec for the instance most
-    /// recently wrapped by this `Vm`.
-    ///
-    /// # Returns
-    ///
-    /// - `Some` if the VM has been created.
-    /// - `None` if no VM has ever been created.
-    pub(super) async fn v0_get(&self) -> Option<InstanceSpecGetResponseV0> {
```

Comment:
This code becomes unnecessary now that the response type has a `From` implementation.